index.html

```diff
@@ -0,0 +1,193 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+
+    <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png" />
+    <script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
+    <link rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css'>            
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" type="text/css">
+    <link rel="stylesheet" href="style.css" />
+    
+    <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap"
+        rel="stylesheet"
+      />
+      <link
+        rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+        integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+        crossorigin="anonymous"
+        referrerpolicy="no-referrer"
+      />
+      <script
+        async
+        src="https://www.googletagmanager.com/gtag/js?id=GA_MEASUREMENT_ID"
+      ></script>
+      
+    <title>Jugoslav Capstone Project 1</title>
+</head>
+<body>
+<!-- -------------------------Header section Starts------------------------------>
+<nav class="header-one">
+    <ul class="padding">
+        <li><i class="fab fa-facebook-f"></i></li>
+        <li><i class="fab fa-twitter"></i></li>
+        <li>english</li>
+        <li>my page</li>
+        <li>logout</li>
+    </ul>
+</nav>
+
+   <!--------Header two---------------------------->
+   <header class="header-two">
+    <nav class="logo">
+        <ul>
+            <li>
+                <a href="index.html"><img src="images/logo.png" alt=""></a>
+            </li>
+        </ul>
+        <span>MilCon 2023</span>
+    </nav>
+   <div id="icon" class="icon fas fa-bars"></div> 
+   
+
+    <nav class="navbar">
+        <ul>
+            <li><a href="about.html" class="active">about</a></li>
+            <li><a href="#">program</a></li>
+            <li><a href="#">join</a></li>
+            <li><a href="#">sponsor</a></li>
+            <li><a href="#">news</a></li>
+            <li class="selected"><a href="">CC campaign</a></li>
+        </ul>
+    </nav>
```

Comment:
- [ ] [**OPTIONAL**]Great job :wave:  addressing all the changes that my fellow have suggested :technologist: , but I noticed that when  the user clicks (or taps) any of the nav link options there is nothing that changes. Kindly make sure that when you click or (taps) :computer_mouse: , any of these links a correct part of the page is displayed.

![Screenshot from 2023-01-07 02-55-48](https://user-images.githubusercontent.com/80458199/211123987-b0e2fa97-031a-4d76-8db7-f067c349300d.png)